.gitignore

```diff
@@ -27,3 +27,9 @@ bytebase-build
 .idea
 
 .DS_Store
+
```

Comment:
Revert this change? See updated readme for starting backend and pull the latest main.